tests/sqlparser_mysql.rs

```diff
@@ -1327,6 +1327,55 @@ fn parse_priority_insert() {
     }
 }
 
+#[test]
+fn parse_insert_as() {
+    let sql = r"INSERT INTO `table` (`date`) VALUES ('2024-01-01') AS `alias` (`mek`)";
```

Comment:
I think it would also be worth testing
1.  the case when there is more than one column alias
2. the case when there is only a row_alias and not any column aliases

Reply:
Tests were fixed.